climada/hazard/centroids/centr.py

```diff
@@ -819,7 +819,10 @@ def set_area_pixel(self, min_resol=1.0e-8, scheduler=None):
             res = np.abs(res).min()
         self.set_geometry_points(scheduler)
         LOGGER.debug('Setting area_pixel %s points.', str(self.lat.size))
-        xy_pixels = self.geometry.buffer(res / 2).envelope
+        xy_pixels = self.to_crs(crs=3857)
+        xy_pixels = self.geometry.buffer(res / 2)
+        xy_pixels = self.to_crs(crs=4326)
```

Comment:
`Centroids` has no method `to_crs`. You need to call this on `geometry`. Also, what do the argument mean? Which coordinate system(s) have you chosen to project the data and why?
```suggestion
        xy_pixels = self.geometry.to_crs(crs=3857)
        xy_pixels = self.geometry.buffer(res / 2)
        xy_pixels = self.geometry.to_crs(crs=4326)
```

Reply:
3857 is apparently a crs that has the units meters, with which the buffer can be calculated. 4326 is our default crs, I suppose? But if I understood it wrong and it doesn't make sense, I'll just close this PR. Thank you for your suggestions!